src/status_im/ui/components/list/views.cljs

```diff
@@ -217,13 +217,15 @@
 (defn flat-list
   "A wrapper for FlatList.
    See https://facebook.github.io/react-native/docs/flatlist.html"
-  [{:keys [data] :as props}]
-  {:pre [(or (nil? data)
-             (sequential? data))]}
-  [flat-list-class
-   (merge (base-list-props props)
-          props
-          {:data (wrap-data data)})])
+  ([props] (flat-list props nil))
+  ([{:keys [data] :as props} {:keys [animated?]}]
```

Comment:
why don't you want just to add animated to props?

Reply:
@flexsurfer the idea is to have another map which is not passed to the chosen react class. maybe `:animated?` do not effect the behaviour but in general it is not always the case...